src/Show.jl

```diff
@@ -307,9 +307,25 @@ function chardeg(t::Table, char::Int64)
 	if char > chartypes(t)
 		throw(DomainError(char, "Character type is out of range."))
 	end
-	t[char].degree
+	return chardeg(t[char])
 end
 
+@doc raw"""
+    chardeg(char::AbstractGenericCharacter)
```

Comment:
Can't we just call this "degree" ?

Reply:
Yes, actually I had it this way before but I thought this could be too much change at once. I'll fix it.